mnist/README.md

```diff
@@ -216,51 +136,56 @@ argo submit model-train.yaml -n ${NAMESPACE} --serviceaccount tf-user \
 Your training workflow should now be executing.
 
 You can verify and keep track of your workflow using the argo commands:
+
 ```
-$ argo list
+$ argo -n ${NAMESPACE} list
 NAME                STATUS    AGE   DURATION
 tf-workflow-h7hwh   Running   1h    1h
 
-$ argo get tf-workflow-h7hwh
+$ argo -n ${NAMESPACE} get tf-workflow-h7hwh
 ```
 
-## Monitoring
+After the STATUS to `Succeeded`, then you can use it.
 
-There are various ways to monitor workflow/training job. In addition to using `kubectl` to query for the status of `pods`, some basic dashboards are also available.
 
-### Argo UI
+## 9.Submit serving workflow[optional]
 
-The Argo UI is useful for seeing what stage your worfklow is in:
+**NOTE: Please only run this when you disable model serving in step 7.**
 
 ```
-PODNAME=$(kubectl get pod -l app=argo-ui -n${NAMESPACE} -o jsonpath='{.items[0].metadata.name}')
-kubectl port-forward ${PODNAME} 8001:8001
+argo -n ${NAMESPACE} list # get the workflow name
+WORKFLOW=<the workflow name>
+argo submit model-deploy.yaml -n ${NAMESPACE} -p workflow=${WORKFLOW} --serviceaccount=tf-user
 ```
 
-You should now be able to visit [http://127.0.0.1:8001](http://127.0.0.1:8001) to see the status of your workflows.
-
-### Tensorboard
+## 10.Using Tensorflow serving
 
-Tensorboard is deployed just before training starts. To connect:
+### Install client requirements
 
 ```
-PODNAME=$(kubectl get pod -l app=tensorboard-${JOB_NAME} -o jsonpath='{.items[0].metadata.name}')
-kubectl port-forward ${PODNAME} 6006:6006
+apt install python-pip python-setuptools --no-install-recommends
```

Comment:
Why do you need to install things on the client?

Reply:
because the cli and webapp depend on them.